contentctl/objects/lookup.py

```diff
@@ -32,6 +34,11 @@ class Lookup(SecurityContentObject):
     match_type: Optional[str] = None
     min_matches: Optional[int] = None
     case_sensitive_match: Optional[bool] = None
+    # TODO: Add id field to all lookup ymls
```

Comment:
Since these are not required or used, we give them default values here.

Reply:
Covered this earlier- makes sense for the "hopefully someday" deal.